NuRadioMC/utilities/medium.py

```diff
@@ -44,6 +45,127 @@ def get_ice_model_radiopropa(self):
             return ice
 """
 
+
+
+class birefringence_index:
```

Comment:
The class shoudl inherit from at least 'IceModel' to make sure it is seen as an icemodel in the code. Also, I think the name should be more specific since it is a very specific ice model (birefringence of southpole 2015 using Jordan et.al. results)